flexget/components/emby/emby_util.py

```diff
@@ -0,0 +1,146 @@
+SCHEMA_SERVER = {
+    'oneOf': [
+        {
+            'type': 'object',
+            'properties': {
+                "protocol": {'type': 'string', 'enum': ['http', 'https'], "default": "http"},
```

Comment:
Having a separate protocol key for this is odd, why not take the protocol from url/address like usually?

Reply:
I guess I agree with you! No need to have this separated! I was mimicking the emby configuration screens, and kept that to the end! Having all in the host will save us 2 lines in the configuration.

I will review this today and push the changes. Since I have a shared schema to all the plugins, the change will be simple.

Reply:
Done @paranoidi I also made some improvements to the media matching. It's good for review and consider merge